scripts/expand_tags.py

```diff
@@ -18,7 +18,11 @@ def main():
                               "be seperated by commas without spaces.\n"))
     args = parser.parse_args()
 
-    m = Mesh(structured=True, mesh=args.mesh_file)
+    try:
+        m = Mesh(structured=True, mesh=args.mesh_file)
+    except:
+        print ("Structured mesh not found, trying unstructured mesh")
```

Comment:
This should probably be a warning, or at least go to stderr, rather than being a print.